python/pyspark/ml/tuning.py

```diff
@@ -346,10 +346,11 @@ class CrossValidatorModel(Model, ValidatorParams, MLReadable, MLWritable):
     .. versionadded:: 1.4.0
     """
 
-    def __init__(self, bestModel):
+    def __init__(self, bestModel, avgMetrics):
```

Comment:
I just realized we should add a default value for avgMetrics since it is technically a public API.  How about a value of ```[]``` by default?